modules/core/04-channel/keeper/packet.go

```diff
@@ -33,6 +33,13 @@ func (k Keeper) SendPacket(
 		return 0, errorsmod.Wrap(types.ErrChannelNotFound, sourceChannel)
 	}
 
+	if channel.FlushStatus != types.NOTINFLUSH {
```

Comment:
ah, nice solution. do you think we still need the `OR` operator/condition below then?

Reply:
Since we are asserting that the channel must be in `OPEN` this check is not strictly necessary (it shouldn't be possible to be in OPEN and not be in `NOTINFLUSH`, but I'm okay being extra defensive. I have a minor preference for the OPEN check to come first (since it is the one which would likely fail)